shared/src/metabase/types.cljc

```diff
@@ -251,29 +251,29 @@
   ;; that's an intentional mapping or not. But it does mean that lots of extra columns will be offered a conversion
   ;; (think Price being offerred to be interpreted as a date)
   (let [numeric-types [:type/BigInteger :type/Integer :type/Decimal]]
-    (reduce #(assoc %1 %2 {:Coercion/UNIXMicroSeconds->DateTime :type/DateTime
-                           :Coercion/UNIXMilliSeconds->DateTime :type/DateTime
-                           :Coercion/UNIXSeconds->DateTime      :type/DateTime})
+    (reduce #(assoc %1 %2 {:Coercion/UNIXMicroSeconds->DateTime :type/Instant
+                           :Coercion/UNIXMilliSeconds->DateTime :type/Instant
+                           :Coercion/UNIXSeconds->DateTime      :type/Instant})
             {:type/Text   {:Coercion/ISO8601->Date     :type/Date
                            :Coercion/ISO8601->DateTime :type/DateTime
                            :Coercion/ISO8601->Time     :type/Time}}
             numeric-types)))
 
-(defn ^:export is_coerceable
+(defn ^:export is-coerceable
   "Returns a boolean of whether a field base-type has any coercion strategies available."
   [base-type]
   (boolean (contains? coercions (keyword base-type))))
 
-(defn ^:export effective_type_for_coercion
+(defn ^:export effective-type-for-coercion
```

Comment:
the JS versions of the functions automatically get converted to snake case, so no need to do it in the source... it looks weird in the Clj code

Reply:
That's true. But using this version makes it clear its intended to be called from js and gives a literal that people can expect to use rather than being surprised or required to know cljs munging rules

Reply:
reverted